subgraphs/eigenda-operator-state/networks.json

```diff
@@ -1,104 +1,88 @@
 {
   "devnet": {
-    "BLSRegistryCoordinatorWithIndices": {
+    "RegistryCoordinator": {
       "address": "0x0000000000000000000000000000000000000000",
       "startBlock": 0
     },
-    "BLSPubkeyRegistry": {
+    "BLSApkRegistry": {
       "address": "0x0000000000000000000000000000000000000000",
       "startBlock": 0
     },
-    "BLSPubkeyCompendium": {
+    "BLSApkRegistry_Operator": {
       "address": "0x0000000000000000000000000000000000000000",
       "startBlock": 0
     },
-    "BLSPubkeyCompendium_Operator": {
+    "RegistryCoordinator_Operator": {
       "address": "0x0000000000000000000000000000000000000000",
       "startBlock": 0
     },
-    "BLSRegistryCoordinatorWithIndices_Operator": {
-      "address": "0x0000000000000000000000000000000000000000",
-      "startBlock": 0
-    },
-    "BLSPubkeyRegistry_QuorumApkUpdates": {
+    "BLSApkRegistry_QuorumApkUpdates": {
       "address": "0x0000000000000000000000000000000000000000",
       "startBlock": 0
     }
   },
   "anvil": {
-    "BLSRegistryCoordinatorWithIndices": {
+    "RegistryCoordinator": {
```

Comment:
What about the addresses of these new contracts?

Reply:
These are dummy addresses that get replaced by the deployer. Making that explicit by zeroing them out. 

It looks like the `preprod-goerli` and `goerli` ones may correspond to actual contracts on goerli, but we can't update those until the new deployment.